Framework/PythonInterface/mantid/api/src/Exports/Algorithm.cpp

```diff
@@ -90,6 +91,26 @@ void translateCancel(const Algorithm::CancelException &exc) {
   PyErr_SetString(PyExc_KeyboardInterrupt, "");
 }
 
+/**
+ * @brief Map an H5::Exception to a Python RuntimeError. Its error stack will be reported
+ *
+ * @param exc - The caught H5::Exception. Its error stack is reported
+ */
+void translateH5Exception(const H5::Exception &exc) {
+  auto handleFrame = [](unsigned int n, const H5E_error_t *err, void *ss_) -> herr_t {
+    std::stringstream &strstream = *((std::stringstream *)ss_);
```

Comment:
`strstream` is a dangerous variable name. There was a good long time when that was an alternate implementation of `stringstream`. Please change this name.

Reply:
`ss_ -> ss_arg`
`strstream -> ss_`